include/bluetooth/iso.h

```diff
@@ -577,6 +577,23 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  */
 int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf);
 
+/** @brief Configure vendor ISO data path
+ *
+ * Request the Controller to configure the data transport path in a given direction between
+ * the Controller and the Host.
+ *
+ *  @param data_path_dir  Direction to be configured, BT_HCI_DATAPATH_DIR_HOST_TO_CTLR or
+ *                        BT_HCI_DATAPATH_DIR_CTLR_TO_HOST
+ *  @param data_path_id   Vendor specific logical transport channel ID, range
+ *                        [BT_HCI_DATAPATH_ID_VS..BT_HCI_DATAPATH_ID_VS_END]
+ *  @param vs_config_len  Length of additional vendor specific configuration data
+ *  @param vs_config      Pointer to additional vendor specific configuration data
+ *
+ *  @return Bytes sent in case of success or negative value in case of error.
+ */
+int bt_iso_configure_data_path(uint8_t data_path_dir, uint8_t data_path_id,
+			       uint8_t vs_config_len, uint8_t *vs_config);
+
```

Comment:
Without being an expert on this particular command, it seems to me that it is a generic, and not an ISO command, although the only use of it seems to be for the ISO data path. 

I'm unsure whether it belongs in iso.h.

Reply:
I think this is the correct place, not sure where else to put it.

Reply:
One possible alternative is to define the function in `bluetooth.h` and move the implementation to `hci_core.c` (and remove the `iso` infix).

Reply:
@jhedberg, @joerchan or @alwa-nordic might have some input here as well.